katzenmint/config/config.go

```diff
@@ -120,7 +121,7 @@ func Load(b []byte) (*Config, error) {
 		return nil, err
 	}
 	if undecoded := md.Undecoded(); len(undecoded) != 0 {
-		return nil, fmt.Errorf("config: Undecoded keys in config file: %v", undecoded)
+		return nil, fmt.Errorf("undecoded keys in config file (%v)", undecoded)
```

Comment:
Which keys is this error message referring to? I would worry about publishing keys via the log errors.

Reply:
User might set the key we don't support in configuration toml file, eg. `nonexist=true`. I think if we print out the key, they would know which key should be removed from toml file.

Reply:
So, you mean command line flag. In that case, this is looks good to me.